usagov/usagov-2021
# Usagov 1305 secondary button padding

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1305

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- This has a few things to check. 

1. All buttons below should have a font size of 1.1REM 
  1.1.  50 state pages
  1.2.  Search Bar Submit 
  1.3. Footer Sign Up English
  1.4. Footer Sign Up Spanish
  1.5. Wizard start button
  1.6. Wizard step before
  1.7. Wizard step next
  1.8. Site Report Issue English
  1.9. Site Report Issue Spanish
  1.10. CEO English
  1.11. CEO Spanish
  1.12. Body Buttons (i.e /food-stamps) 

2. Light Blue Background Color applied for correct buttons (class="usa-button--secondary")
  2.1.  50 state pages
  2.2.  ~~Search Bar Submit~~
  2.3. Wizard start button
  2.4. Wizard step next
  2.5. Site Report Issue English
  2.6. Site Report Issue Spanish
  2.7. CEO English
  2.8. CEO Spanish
  2.9.  Body Buttons (i.e /food-stamps) 

3. Larger padding applied for most buttons (class="usa-button--big")
  3.1. Wizard start button
  3.2. Wizard step before
  3.3. Wizard step next
  3.4. Site Report Issue English
  3.5. Site Report Issue Spanish
  3.6. CEO English
  3.7. CEO Spanish
  3.8.  Body Buttons (i.e /food-stamps) 

4. Note these buttons have special paddings to keep them in line with other elements: 
  4.1.  50 state pages
  4.2.  Search Bar Submit 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
7. Find the commit of this pull request.
8. Build and deploy the changes.
10. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
